wa/workloads/uibench/__init__.py

```diff
@@ -0,0 +1,60 @@
+#    Copyright 2017 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from time import sleep
+
+from wa import ApkWorkload, Parameter
+
+class UiBench(ApkWorkload):
+
+    name = 'uibench'
+    description = """
+    Benchmark from AOSP for evaluating performance of Android UI elements.
+
+    Built from AOSP source with 'make UiBench'. Tested with AOSP branch
+    android-8.0.0_r4.
+    """
+    package = 'com.android.test.uibench'
+
+    test_activities = [
+        'BitmapUploadActivity',
+        'DialogListActivity',
+        'EditTextTypeActivity',
+        'FullscreenOverdrawActivity',
+        'GlTextureViewActivity',
+        'InflatingListActivity',
+        'InvalidateActivity',
+        'ShadowGridActivity',
+        'TextCacheHighHitrateActivity',
+        'TextCacheLowHitrateActivity',
+        'ActivityTransition',
+        'ActivityTransitionDetails',
+        'TrivialAnimationActivity',
+        'TrivialListActivity',
+        'TrivialRecyclerViewActivity',
+    ]
+
+    parameters = [
+        Parameter('activity',
+                  mandatory=True,
```

Comment:
Mandatory parameters should be avoided unless absolutely necessary -- pick on of the available activities as the default.